datascience/predicates.py

```diff
@@ -211,6 +211,49 @@ def not_containing(substring):
     def not_contained_in(superstring):
         """A string that is not contained within the superstring"""
         return -(are.contained_in(superstring))
+
+    ###############
+    #   Aliases   #
+    ###############
+
+    @staticmethod
+    def less_than(y):
+        """Less than y."""
+        return are.below(y)
+
+    @staticmethod
+    def greater_than(y):
+        """Greater than y."""
+        return are.above(y)
+
+    @staticmethod
+    def greater_than_or_equal_to(y):
+        return are.above_or_equal_to(y)
+
+    @staticmethod
+    def less_than_or_equal_to(y):
+        return are.below_or_equal_to(y)
```

Comment:
Based on: https://coveralls.io/builds/55559799/source?filename=datascience%2Fpredicates.py

We're missing test coverage on this method and a few others. The tests you've created are fantastic so far, if you could please add a few more for the missing test methods - this should be mergeable!